test/blackbox-tests/useCallback.t

```diff
@@ -0,0 +1,53 @@
+
+  $ cat >dune-project <<EOF
+  > (lang dune 3.9)
+  > (using melange 0.1)
+  > EOF
+
+  $ cat >dune <<EOF
+  > (melange.emit
+  >  (target js-out)
+  >  (emit_stdlib false)
+  >  (preprocess (pps melange.ppx))
+  >  (libraries reason-react))
+  > EOF
+
+  $ cat >x.ml <<EOF
+  > let cb = React.useCallback0(fun a b -> a + b)
```

Comment:
Sorry for late feedback. Just realized: do we want to document the case I mentioned before? the one that produces a curried function in JS: `fun a -> let f b = a + b in f`.

Reply:
I don't think so. That's quite obvious from the way it's written.